### PR TITLE
Add validationMessages field in metadata

### PR DESCRIPTION
**Pull request by kishan3**
Schema: https://docs.google.com/spreadsheets/d/1UBKA73GGl0113z66T-6Ufq9w1fTiRf-dmuVjXqNFEMo/edit#gid=0
TeamWork card: https://zyte.eu.teamwork.com/app/tasks/24805139

### CODE REVIEW

**Review by Gallaecio — APPROVED** [review]

File: zyte_common_items/components.py [hunk 1]
```diff
@@ -38,6 +38,7 @@ class _ListMetadata(Item):
     #: Date and time when the product data was downloaded, in UTC timezone and
     #: the following format: ``YYYY-MM-DDThh:mm:ssZ``.
     dateDownloaded: Optional[str] = None
+    validation: Optional[dict] = None
```
Comment (Gallaecio) [inline zyte_common_items/components.py:41]: Could you add documentation for the field? (with `#:`-prefixed comments like above)

File: zyte_common_items/components.py [hunk 2]
```diff
@@ -39,6 +39,9 @@ class _ListMetadata(Item):
     #: the following format: ``YYYY-MM-DDThh:mm:ssZ``.
     dateDownloaded: Optional[str] = None
 
+    #: Contains paths to fields with the description of issues found with their values.
+    validationMessages: Optional[List[str]] = None
```
Comment (lopuhin) [inline zyte_common_items/components.py:43]: I think it should be ``Optional[Dict[str, List[str]]]`` according to @VMRuiz

**Review by Gallaecio — APPROVED** [review]

**Review by VMRuiz — APPROVED** [review]

**Comment by VMRuiz** [comment]
@Gallaecio Are we good to merge this?

**Comment by Gallaecio** [comment]
Unless @lopuhin wants to have a final look, I think we are good to go, yes.

File: zyte_common_items/components.py [hunk 3]
```diff
@@ -39,6 +39,9 @@ class _ListMetadata(Item):
     #: the following format: ``YYYY-MM-DDThh:mm:ssZ``.
     dateDownloaded: Optional[str] = None
 
+    #: Contains paths to fields with the description of issues found with their values.
+    validationMessages: Optional[Dict[str, List[str]]] = None
```
Comment (kmike) [inline zyte_common_items/components.py:43]: I think we need to updated docstrings for various metadata classes: this one, _DetailsMetadata, maybe something else.

Comment (Gallaecio) [reply zyte_common_items/components.py:43]: Yes. Maybe these docstrings should not mention their fields to begin with.

Comment (Gallaecio) [reply zyte_common_items/components.py:43]: https://github.com/zytedata/zyte-common-items/pull/88/commits/828d9da88d40a001e958d96c0c92bc2e15ab634d ?

**Review by lopuhin — APPROVED** [review]
Thanks @kishan3 👍

**Pull request merged**